source/slang/hlsl.meta.slang

```diff
@@ -20654,7 +20654,7 @@ struct HitObject
         __target_switch
         {
         case hlsl: __intrinsic_asm ".SetShaderTableIndex";
-        case cuda: __intrinsic_asm "optixHitObjectSetSbtRecordIndex";
+        case cuda: __intrinsic_asm "optixHitObjectSetSbtRecordIndex($1)";
```

Comment:
`optixHitObjectSetSbtRecordIndex` takes a parameter `sbtRecordIndex`, but `LoadLocalRootTableConstant`'s parameter is `byteOffset`. 

`byteOffset` is not `record index`, so you need to convert it.

Reply:
Ah, nevermind, I thought this is inside `LoadLocalRootTableConstant`, but this is `GetShaderTableIndex`, so this is fine.